components/config/theme.ts

```diff
@@ -2,38 +2,62 @@ import { TextStyle } from 'react-native'
 
 export const colors = {
   dark: {
-    neutral: '#EAEADF',
-    cta: '#398FE4',
-    background: '#13141C',
-    cards: '#161E2C',
-    online: '#0A741C',
-    text: '#aaa',
+    // Brand colors
+    sand: '#EAEADF',
+    gravity: '#398FE4',
+    bee: '#FFDF41',
+    nebula: '#923589',
+
+    // Primary Colors
+    primarySand: '#EAEADF',
+    primarySand60: '#EAEADF99',
+    primarySand40: '#EAEADF66',
```

Comment:
@J-Rehan shouldn’t the last two digits be 60 & 40 respectively? I think that’s how you set opacity but maybe i have it wrong

Reply:
Since we are using hex code the value ranges from **0-F** and `60%` would be equivalent to `99` in hex.

Here's the documentation for it: https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4
Also, we can use this online tool to cover **RGBA** values to **hex** values: https://rgbacolorpicker.com/rgba-to-hex